docs/sources/alerting/unified-alerting/message-templating.md

```diff
@@ -0,0 +1,168 @@
++++
+title = "Message templating"
+description = "Message templating"
+keywords = ["grafana", "alerting", "guide", "contact point", "templating"]
+weight = 400
++++
+
+# Message templating
+
+Notifications sent via [contact points]({{< relref "./contact-points.md" >}}) are built using templates. Grafana comes with default templates, but they can also be customized. The Grafana's notification templates are based on the [Go templating system](https://golang.org/pkg/text/template). Note that some fields are evaluated as text, and others as HTML which will affect escaping. Most of contact point fields can be templated, and reusable templates can be created and used in multiple contact points.
```

Comment:
```suggestion
Notifications sent via [contact points]({{< relref "./contact-points.md" >}}) are built using templates. Grafana comes with default templates which you can customize. Grafana's notification templates are based on the [Go templating system](https://golang.org/pkg/text/template) where some fields are evaluated as text, while others are evaluated as HTML which can affect escaping. Since most of the contact point fields can be templated, you can create reusable templates and them in multiple contact points.
```